### ISSUE TITLE
Parse failsures when Vaisala indicates invalid data.

### ISSUE THREAD

**Issue by pchote** [warwick-one-metre/vaisalad#12]
Data:

```
0R1,Dn=334#,Dm=338#,Dx=345#,Sn=50.6#,Sm=52.3#,Sx=55.4#
0R5,Th=0.9C,Vh=18.6V,Vs=12.2V,Vr=3.522V
0R0,Dm=338#,Sm=52.3#,Ta=-1.1C,Ua=88.0P,Pa=769.3H,Rc=63.69M,Th=0.9C,Vh=18.6V
0R2,Ta87.9P,Pa=769.3H
0R1,Dn=334#,Dm=338#,Dx=345#,Sn=50.6#,Sm=52.3#,Sx=55.4#
0R0,Dm=338#,Sm=52.3#,Ta=-1.1C,Ua=87.9P,Pa=769.3H,Rc=63.69M,Th=0.9C,Vh=18.6V
0R3,Rc=63.69M,Rd=22280s,Ri=0.0M,Hc=0.0M,Hd=40s,Hi=0.0M
0R2,Ta=-1.1C,Ua=88.0P,Pa=769.2H
0R1,Dn=334#,Dm=338#,Dx=345#,Sn=50.6#,Sm=52.3#,Sx=55.4#
0R0,Dm=338#,Sm=52.3#,Ta=-1.1C,Ua=88.1P,Pa=769.2H,Rc=63.69M,Th=0.9C,Vh=18.6V
0R2,Ta=-1.1C,Ua=88.1P,Pa=769.2H
0R1,Dn=334#,Dm=338#,Dx=345#,Sn=50.6#,Sm=52.3#,Sx=55.4#
0R5,Th=0.7C,Vh=12.1V,Vs=12.3V,Vr=3.522V
0R0,Dm=338#,Sm=52.3#,Ta=-1.1C,Ua=88.1P,Pa=769.2H,Rc=63.69M,Th=0.7C,Vh=12.1V
```

The status codes for wind has changed to `#`.  Manual says this means the measurement is invalid (but doesn't indicate why).  All fields except Vs and Vr can potentially take `#`.

**Comment by pchote** [comment]
Can deal with these states by adding a new output var with status flags for each value, and then set the observing condition to false in environmentd when the value becomes non-zero.  This avoids having to deal with missing measurements for individual parameters.

**Comment by pchote** [comment]
Fixed in a6554edfe7ee8a8bebb0b4a22c1172eb16a365f5.

`environmentd` will need to be updated to use the new `_valid` members.
